customChatWidget/README.md

```diff
@@ -139,6 +139,8 @@ Below steps explain how Custom Chat Widget works on a web page
 					sampleKey1: 'sampleValue1',
 					sampleKey2: 'sampleValue2',
 				},
```

Comment:
[nit] Indentation!

Reply:
Used spaces not tabs, code editor doesn't pick up. Fixed in next revision